### PR TITLE
Fix tabs theming

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/IW-4241

### CODE REVIEW

File: style-guide/styles/wds-components/_tabs.scss [hunk 1]
```diff
@@ -35,6 +35,7 @@
 
 		> a {
 			align-items: center;
+			color: inherit;
```
Comment (latata) [inline style-guide/styles/wds-components/_tabs.scss:38]: let's inherit the color from the tab's label element

File: style-guide/styles/wds-mixins/_tabs-theming.scss [hunk 2]
```diff
@@ -14,21 +14,13 @@
 		}
 
 		&.wds-is-current {
-			@include wds-box-underline($tab-color);
+			@include wds-box-underline($tab-hover-color);
 		}
 
-		&.wds-is-active,
```
Comment (latata) [inline style-guide/styles/wds-mixins/_tabs-theming.scss:20]: `.wds-is-active` is not used with tabs

File: style-guide/styles/wds-mixins/_tabs-theming.scss [hunk 3]
```diff
@@ -14,21 +14,13 @@
...
 			.wds-tabs__tab-label {
 				color: $tab-hover-color;
-
-				> a {
-					color: $tab-hover-color;
-				}
-			}
-
-			&.wds-is-current {
-				@include wds-box-underline($tab-hover-color);
```
Comment (latata) [inline style-guide/styles/wds-mixins/_tabs-theming.scss:31]: underline only for active tab

File: dist/scss/wds-components/_tabs.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .wds-tabs {
-	@include tabs-theming($wds-fandom-color-link, $wds-fandom-color-link-hover);
+	@include tabs-theming($wds-color-black, $wds-fandom-color-link);
```
Comment (ustczanka) [inline dist/scss/wds-components/_tabs.scss:2]: if it's for text color, it should rather be $wds-fandom-color-black

**Review by ustczanka — APPROVED** [review]

**Pull request merged**